src/transaction.ts

```diff
@@ -11,13 +11,22 @@ export class Transaction implements IModel  {
   public amount: number;
   public date: Date;
   public signature: string;
+  public smartContractId: number;
```

Comment:
I'm not sure on this, but I think the ID will be of type GUID/ UUID. Because this type does not exists in typescript use the type `string`.